public/index.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
```

Comment:
![image](https://github.com/lorenzoworx/math/assets/93039359/8f95ed80-b4f7-4cc2-8af7-634cddb37a11)

- [ ] Kindly fix your MIT link to make your readme descriptive and professional :+1: It is broken because right now you don't have any MIT file in your project :fearful: Kindly make an MIT file in your project to make your link work and make your project licensed :+1: